views/beneficial-owner-gov.html

```diff
@@ -0,0 +1,322 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Government or public authority beneficial owner - Register an overseas entity - GOV.UK
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the government or public authority beneficial owner</h1>
+      <p class="govuk-caption-l  govuk-!-margin-bottom-5 govuk-!-margin-top-5">You can add more government or public authority beneficial owners later.</p>
+
+      <form class="form" method="post">
+
+        {{ govukDetails({
+          summaryText: "What is a government or public authority beneficial owner?",
+          text: "Here"
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: "What is its name?",
+            classes: "govuk-label--m",
+            isPageHeading: false
+          },
+          id: "corpName",
+          name: "corpName"
+        }) }}
+        {% call govukFieldset({
+          legend: {
+            text: "What is its principle or registered office address?",
+            classes: "govuk-fieldset__legend--m",
+            isPageHeading: false
+          }
+        }) %}
+
+        {{ govukInput({
+          label: {
+            html: 'Property name or number (optional)'
+          },
+          hint: {
+           text: "For example, 'The Mill', '116' or 'Flat 37a'"
+         },
+          id: "principleAddressPropertyNameNumber",
+          name: "principleAddressPropertyNameNumber"
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: "Address line 1"
+          },
+          classes: "govuk-!-width-two-thirds",
+          id: "principleAddressLine1",
+          name: "principleAddressLine1",
+          autocomplete: "addressLine1"
```

Comment:
Think we said we'd keep this consistent with the id/name - so principleAddressLine1.